production/ksonnet/loki/shipper.libsonnet

```diff
@@ -25,20 +25,20 @@
     index_period_hours: if self.using_shipper_store then 24 else super.index_period_hours,
     loki+: if self.using_shipper_store then {
       storage_config+: {
-        boltdb_shipper+: {
-          shared_store: $._config.boltdb_shipper_shared_store,
-          active_index_directory: '/data/index',
-          cache_location: '/data/boltdb-cache',
-        },
-        tsdb_shipper+: {
-          shared_store: $._config.tsdb_shipper_shared_store,
-          active_index_directory: '/data/tsdb-index',
-          cache_location: '/data/tsdb-cache',
-        },
+        boltdb_shipper+: if $._config.using_tsdb_shipper then ({
```

Comment:
```suggestion
        boltdb_shipper+: if $._config.using_boltdb_shipper then ({
```

Reply:
Also, I think a `tk fmt` would make this more readable.

Reply:
I tried tk fmt on this, but it just said files are already formatted.

Reply:
Removing the parentheses and doing a `tk fmt` seems to format it properly.